File: CodeFreak1/ViewModel/SubmissionViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace CodeFreak1.ViewModel
{
    public class SubmissionViewModel : RequestStatus
    {
        public string SubmissionId { get; set; }
        public string UserId { get; set; }
        public string ProblemId { get; set; }
        public int Score { get; set; }
        public string Status { get; set; }
        public DateTime SubmissionDateTime { get; set; }
        public string Code { get; set; }
        public int LanguageId { get; set; }

        public List<SubmissionProblemTestCaseViewModel> submissionProblemTestCase { get; set; }
    }
}

File: CodeFreak1/ClientApp/src/app/problem/dtos/submission-view-model.ts
import { RequestStatus } from "../../Security/Dtos/request-status";
import { SubmissionProblemTestCaseViewModel } from "./submission-problem-testcase-view-model";

export class SubmissionViewModel extends RequestStatus {

  public SubmissionId: string;
  public UserId: string;
  public ProblemId: string;
  public Score: number;
  public Status: string;
  public SubmissionDateTime: string;
  public Code: string;

  public submissionProblemTestCase: Array<SubmissionProblemTestCaseViewModel>;
}
